src/core/lib/iomgr/exec_ctx.cc

```diff
@@ -104,11 +104,23 @@ static grpc_closure_scheduler exec_ctx_scheduler = {&exec_ctx_scheduler_vtable};
 grpc_closure_scheduler* grpc_schedule_on_exec_ctx = &exec_ctx_scheduler;
 
 namespace grpc_core {
-GPR_TLS_CLASS_DEF(ExecCtx::exec_ctx_);
+namespace {
+GPR_TLS_DECL(g_exec_ctx);
```

Comment:
I know that at the current time, this is doubling-down since GPR_TLS_DECL already gives it static linkage but I thought that I should wrap it in an anonymous namespace to make the intention more clear.